iwyu_path_util.cc

```diff
@@ -177,18 +189,27 @@ string ConvertToQuotedInclude(const string& filepath) {
   for (Each<HeaderSearchPath> it(&search_paths); !it.AtEnd(); ++it) {
     // All header search paths have a trailing "/", so we'll get a perfect
     // quoted include by just stripping the prefix.
-    if (StripLeft(&path, it->path)) {
-      if (it->path_type == HeaderSearchPath::kSystemPath)
-        return "<" + path + ">";
-      else
-        return "\"" + path + "\"";
-    }
+
+    if (!StripPathPrefix(&path, it->path))
```

Comment:
I think this block was clearer before, i.e. something like:

    // All header search paths have a trailing "/", so we'll get a perfect
    // quoted include by just stripping the prefix.
    if (StripPathPrefix(&path, it->path)) {
      if (it->path_type == HeaderSearchPath::kSystemPath)
        return "<" + path + ">";
      else
        return "\"" + path + "\"";
    }

The `continue` and if/else made it harder to follow.